sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -874,4 +876,47 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     checkAnswer(df, Seq(Row(2)))
     // scalastyle:on
   }
+
+  private def checkAverageConversion(df: DataFrame): Unit = {
```

Comment:
this does not check the partial agg pushdown. I think we need to test two cases:
1. the JDBC table has only 1 partition and supports complete pushdown, so there is no agg operator at Spark side
2. the JDBC table doesn't support complete pushdown, there is only one agg operator at Spark side as the partial agg has been pushed.